ssr/src/page/token/sns_form.rs

```diff
@@ -8,8 +8,7 @@ use sns_validation::{
     humanize::{parse_duration, parse_percentage, parse_tokens},
     pbs::nns_pb,
 };
-
-use crate::state::canisters::Canisters;
+use yral_canisters_common::Canisters;
 
 #[derive(Clone, Debug)]
 struct NeuronForm {
```

Comment:
Is it possible to refactor this too as it contains a lot of default values...

Same goes for other form states in this file too...

Reply:
these states are directly linked with the UI and the state code for mobile apps will be entirely different. We can move it to a separate crate but doesnt make sense to put them in the common one